app/containers/NewMediaCall/NewMediaCall.integration.test.tsx

```
@@ -0,0 +1,407 @@
+import { useCallStore } from '../../lib/services/voip/useCallStore';
+import { mediaSessionInstance } from '../../lib/services/voip/MediaSessionInstance';
+import { mockedStore } from '../../reducers/mockedStore';
+import type { TPeerItem } from '../../lib/services/voip/getPeerAutocompleteOptions';
+import type { InsideStackParamList } from '../../stacks/types';
+
+// Compile-time route-name guard — renaming 'CallView' in the stacks breaks tsc.
+// (CallView: undefined is defined at app/stacks/types.ts:299)
+// `void` prevents the noUnusedLocals error; the type annotation is the actual guard.
+type AssertCallViewRoute = InsideStackParamList extends { CallView: unknown } ? true : never;
+const _routeCheck: AssertCallViewRoute = true;
+void _routeCheck;
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)